ui/components/app/connected-accounts-permissions/connected-accounts-permissions.js

```diff
@@ -55,7 +55,7 @@ const ConnectedAccountsPermissions = ({ permissions }) => {
                 className="connected-accounts-permissions__checkbox"
               />
               <label htmlFor={permissionName}>
-                {getPermissionDescription(t, permissionName).label}
+                {getPermissionDescription(t, permissionName)[0].label}
```

Comment:
The length of the result should be asserted, and the function should have a `permissionValue` passed to it.

Reply:
Should be fixed with latest commit